packages/article/src/Infrastructure/Symfony/HttpKernel/SuluArticleBundle.php

```diff
@@ -425,8 +425,8 @@ public function loadExtension(array $config, ContainerConfigurator $container, C
             ])
             ->tag('sulu_route.route_defaults_provider', ['resource_key' => 'articles']);
 
-        $services->set('sulu_article.article_index_listener')
-            ->class('Sulu\Article\Infrastructure\Sulu\Search\ArticleIndexListener')
+        $services->set('sulu_article.admin_article_index_listener')
+            ->class('Sulu\Article\Infrastructure\Sulu\Search\AdminArticleIndexListener')
```

Comment:
If you're already replacing it maybe using class strings like "AdminArticleIndexListener::class" Then a rename would have caught that.